src/gpodder/util.py

```diff
@@ -857,3 +857,11 @@ def find_command(command):
 
 def read_json(url):
     return json.loads(urlopen(url).read().decode('utf-8'))
+
+def validate_url(url):
```

Comment:
Is this generic enough that it should be put here? Or should it rather be a private function in `itunes.py`?

It doesn't really do any URL validation, it merely checks if `scheme` and `netloc` exist (at least it being in "util" (kind of high-level) and named "validate_url", I'd expect it to be a bit more involved...).

Would `util.normalize_feed_url()` be a better function to use?